cogs/sports.py

```diff
@@ -657,6 +657,11 @@ async def do_nhl_scores(self, ctx, *, optional_input: str=None):
              nhl --tz pdt bos
         """
 
+        async def fetch_json(self, url: str):
```

Comment:
As with the `mma.py` comment, I think this could be extracted out of the `do_nhl_scores` method and into the class so that the changes below (calling `self.fetch_json()`) will actually work